sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -3066,9 +3066,12 @@ class Analyzer(
             inputAttributes
           }
 
+          // println(s"\n\ndeserializer= $deserializer\n")
+          // println(s"\n\ninputs= $inputs\n")
```

Comment:
Hi, @erikerlandson . Could you clean up these debugging comments?

Reply:
@dongjoon-hyun  yes, I'll definitely clean those up.  Before I do final clean-up I'm interested in getting opinions on whether this particular fix is a good one, or if it should be fixed in some other way.